### PR TITLE
Ag preview trailer

### PR DESCRIPTION
**Pull request by anuragdce**

### CODE REVIEW

File: TVUIComponent/lib/src/main/java/com/amazon/android/uamp/ui/PlaybackTrailerActivity.java [hunk 1]
```diff
@@ -0,0 +1,2241 @@
...
+ * PlaybackOverlayActivity for content playback that loads PlaybackOverlayFragment
+ */
+public class PlaybackTrailerActivity extends Activity implements
+        PlaybackOverlayFragment.OnPlayPauseClickedListener, AMZNMediaPlayer
+        .OnStateChangeListener, AMZNMediaPlayer.OnErrorListener, AMZNMediaPlayer.OnInfoListener,
+        AudioManager.OnAudioFocusChangeListener, AMZNMediaPlayer.OnCuesListener,
+        ErrorDialogFragment.ErrorDialogFragmentListener {
+
+    private static final int TRANSPORT_CONTROLS_DELAY_PERIOD = 50;
+    private static final String TAG = PlaybackActivity.class.getSimpleName();
```
Comment (echedev) [inline TVUIComponent/lib/src/main/java/com/amazon/android/uamp/ui/PlaybackTrailerActivity.java]: I guess we should use `PlaybackTrailerActivity.class` here

File: TVUIComponent/lib/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -30,55 +28,57 @@ permissions and limitations under the License.
...
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.amazon.android.tv.tenfoot"
-          android:versionCode="1"
-          android:versionName="1.1">
+    package="com.amazon.android.tv.tenfoot"
+    android:versionCode="1"
+    android:versionName="1.1">
 
     <uses-sdk
-            android:minSdkVersion="21"
```
Comment (echedev) [inline TVUIComponent/lib/src/main/AndroidManifest.xml]: Could we avoid all those code formatting changes? They will complicate update to new Fire App Builder release in future.

File: ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
+ * http://aws.amazon.com/apache2.0/
+ * <p>
  * or in the "license" file accompanying this file. This file is distributed
  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
 package com.amazon.android.contentbrowser;
 
+import android.app.Activity;
```
Comment (echedev) [inline ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: Could we avoid all those code formatting changes and displacements of original code blocks? They will complicate update to new Fire App Builder release and merging the original code with our changes in future .

**Review by echedev — COMMENTED** [review]
Basically looks good. Playing trailer feature is working.
Please check few comments. Mostly they are on code formatting changes and rearrangements of original Fire Builder code that are not related to trailer feature. I think those changes may complicate future updates to new versions of Fire App Builder. I would try to keep their code unchanged where possible.

File: TVUIComponent/lib/src/main/java/com/amazon/android/uamp/ui/PlaybackTrailerActivity.java (hunk 1 above)
Comment (anuragdce) [reply TVUIComponent/lib/src/main/java/com/amazon/android/uamp/ui/PlaybackTrailerActivity.java]: fixed

File: ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java (hunk 3 above)
Comment (anuragdce) [reply ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java]: please check the latest PR, formatting issues are resolved. thanks

**Review by echedev — APPROVED** [review]

**Review by echedev — APPROVED** [review]

**Pull request merged**